src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.catalog {
+  display: grid;
+  margin: 0;
+
+  &__content {
+    display: grid;
+    margin: 0 auto;
+    padding: 50px 40px;
+    gap: 46px 48px;
+    grid-template-columns: repeat(1, 200px);
+
+    @media (min-width: 488px) {
+      & {
+        grid-template-columns: repeat(2, 200px);
+      }
+    }
+
+    @media (min-width: 768px) {
+      & {
+        grid-template-columns: repeat(3, 200px);
```

Comment:
you can create a variable for this 200px which you write several times (like $card-width)